.github/workflows/publish-main-android.yml

```diff
@@ -0,0 +1,28 @@
+name: Update production android apptp blocklist
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2.1.2
+        with:
+          node-version: "14"
+
+      - name: Push to CDN
+        uses: jakejarvis/s3-sync-action@v0
+        with:
+          args: --acl public-read --follow-symlinks
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          SOURCE_DIR: "app/android-tds.json"
+          DEST_DIR: "trackerblocking/appTP/2.0"
```

Comment:
Just double checking this isn't gonna publish a file to `2.0`, i.e., will actually see this is a directory?

Reply:
It's the same way [it's currently set up](https://github.com/duckduckgo/app-tracking-protection-lists/blob/140908f64150fd6f10abba6a5d8647c3ec70caf3/.github/workflows/publish-main.yml#L28), so I'm assuming it works.

Reply:
Yeah s3 uses a pseudo directory structure, so pushing `trackerblocking/appTP/2.0/blocklist.json` should work even if the `appTP/2.0` directory doesn't currently exist